src/screens/Home/SignUp.jsx

```diff
@@ -159,8 +159,11 @@ const SignUp = () => {
           </div>
         </div>
       </div>
-      <Footer/>
+      <div class="pt-6">
+        <Footer></Footer>
```

Comment:
`<Footer />`

BTW i have this big doubt, why we using Footer on every page differently and why not design it in a generic way of having a Navbar, <MainContent />, Footer in main APP.js and only mainContent because navbar and footer will always remain static??

Correct me if I am not wrong here.